### ISSUE TITLE
create partition topic by function

### ISSUE THREAD

**Issue by JKeyZ** [apache/pulsar#6536]
how can i to create  new patitioned topics by function?

The partition number is defind by user-config by function

**Comment by jiazhai** [comment]
@JKeyZ you can use [pulsar-admin](https://pulsar.apache.org/docs/en/pulsar-admin/#create-partitioned-topic) or rest api to create partitioned topic before starting a Pulsar functions. 
Or, you could reset broker.conf for "auto create partition number"(this is currently broker level setting):
```

# Enable topic auto creation if new producer or consumer connected (disable auto creation with value false)
allowAutoTopicCreation=true

# The type of topic that is allowed to be automatically created.(partitioned/non-partitioned)
allowAutoTopicCreationType=partitioned < ===

# Enable subscription auto creation if new consumer connected (disable auto creation with value false)
allowAutoSubscriptionCreation=true. < ===

# The number of partitioned topics that is allowed to be automatically created if allowAutoTopicCreationType is partitioned.
defaultNumPartitions=1. < === default number you wanted.
```

**Comment by JKeyZ** [comment]
Sorry for replay later，I have a case to proccess binlog of mysql. Some tables need consume sequence (for ex: order info, pay info), some tables consume parallel (for ex: comment info). So I want to create topic as many as I need partitions for per table.

**Comment by tisonkun** [comment]
Closed as stale. Please open a new issue if it's still relevant in maintained versions.
